src/parser/priest/discipline/CombatLogParser.js

```diff
@@ -97,7 +97,7 @@ class CombatLogParser extends CoreCombatLogParser {
     lucidDreams: LucidDreams,
 
     // Covenants
-    mindgames: Mindgames,
+    mindgames: MindGames,
```

Comment:
it's Mindgames, not Mind Games actually.
https://shadowlands.wowhead.com/spell=323706/mindgames